streams-contrib/streams-provider-google/google-gplus/src/test/java/org/apache/streams/gplus/test/processors/GooglePlusTypeConverterIT.java

```diff
@@ -93,15 +93,17 @@ public void testProcessPerson() throws IOException, ActivitySerializerException
         GooglePlusActivityUtil.updateActivity(person, activity);
 
         assertEquals(retList.size(), 1);
-        assert (retList.get(0).getDocument() instanceof Activity);
+        assert(retList.get(0).getDocument() instanceof Activity);
```

Comment:
change all 'assert' to 'Assert.assertXXX()'. this needs to be done here and many other places in the project.